examples/erc20.rs

```diff
@@ -265,6 +265,19 @@ impl StarknetSyscallHandler for SyscallHandler {
     ) -> SyscallResult<(U256, U256)> {
         unimplemented!()
     }
+
+    fn sha256_process_block(
+        &mut self,
+        _prev_state: &[u32; 8],
+        _current_block: &[u32; 16],
+        _remaining_gas: &mut u128,
+    ) -> SyscallResult<[u32; 8]> {
+        unimplemented!()
+    }
+
+    fn cheatcode(&mut self, _selector: Felt, _input: &[Felt]) -> Vec<Felt> {
+        std::unimplemented!();
```

Comment:
Just use `unimplemented!()`. Or even better, don't implement it at all (it should have a default implementation, if I recall correctly).